### PR TITLE
Create a warning for outdated plugins

### PR DESCRIPTION
**Pull request by smartcontracts**
We're planning to icebox a few plugins. The JSvm is extremely nice, but it's too difficult to maintain two different node forks right now. Most contracts shouldn't even need the custom JSvm. We'll reintroduce these plugins eventually (or if community members want to maintain them). Users will be recommended to use our `up.sh` script.

### CODE REVIEW

**Review by tynes — APPROVED** [review]
Looks good to me. Maybe the one thing would be to add is a nudge to find maintainers for the short term

**Comment by tynes** [comment]
@smartcontracts Is this ready to be merged?

**Comment by smartcontracts** [comment]
I actually think we should update our tutorial/docs before making this merge. Will be very confusing for end-users otherwise.

**Pull request closed without merging**
